src/api/Jalasoft.TeamUp.Resumes.API/Controllers/GetResumes.cs

```diff
@@ -21,24 +22,28 @@ public GetResumes(IResumesService resumesService)
         }
 
         [FunctionName("GetResumes")]
-        [OpenApiOperation(operationId: "GetResumes", tags: new[] { "Resumes" })]
+        [OpenApiOperation(operationId: "GetResumes", tags: new[] { "Resumes" })]
+        [OpenApiParameter(name: "skill", In = ParameterLocation.Query, Required = false, Type = typeof(string), Description = "The name of the skill to search by.")]
         [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Resume[]), Description = "Successful response")]
+        [OpenApiResponseWithoutBody(statusCode: HttpStatusCode.NotFound, Description = "Resource not found")]
         public IActionResult Run(
             [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "v1/resumes")] HttpRequest req)
         {
             try
             {
-                var resumes = this.resumesService.GetResumes();
+                string skill = req.Query["skill"];
+                var resumes = this.resumesService.GetResumes(skill);
                 if (resumes == null)
                 {
                     throw new ResumesException(ResumesErrors.NotFound);
                 }
 
                 return new OkObjectResult(resumes);
             }
-            catch (ResumesException e)
+            catch (ResumesException ex)
             {
-                return e.Error;
+                var error = ex.Error;
+                return error;
             }
             catch (System.Exception e)
             {
```

Comment:
Are you overwriting the getResumes service?

Reply:
it's for an endpoint